src/pybind/mgr/prometheus/module.py

```diff
@@ -740,6 +742,13 @@ def _setup_static_metrics(self) -> Dict[str, Metric]:
             RBD_MIRROR_METADATA
         )
 
+        metrics['nvmeof_metadata'] = Metric(
+            'untyped',
+            'nvmeof_metadata',
+            'NVMeoF Metadata',
+            NVMEOF_METADATA
+        )
+
```

Comment:
@VallariAg like NVME can have different kind of metadata. 

NVME gateway group metadata 
NVME gateway metadata.

I think what you are trying to expose if daemon status of gateway, if I'm not wrong. So lets make this status metric rather than metadata, is what I'm trying to say. i.e NVMEOF_GATEWAY_STATUS

In this case the status would be output of the metric and not the label.